src/test/java/com/egov/municipal-services/requestPayload/fsmService/vehicalTripCreateWithoutVehicalId.json

```diff
@@ -0,0 +1,78 @@
+{
```

Comment:
@murali-egov , what about this file. it says vehicleTripCreateWithoutVehicleId. I think this also be removed